examples/candid_types/test/tests.ts

```diff
@@ -0,0 +1,43 @@
+import { Test } from 'azle/test';
+import { _SERVICE } from './dfx_generated/candid_types/candid_types.did';
+import { ActorSubclass } from '@dfinity/agent';
+
+export function getTests(candid_canister: ActorSubclass<_SERVICE>): Test[] {
+    return [
+        {
+            name: 'test query with keyword name',
+            test: async () => {
+                await candid_canister.opt();
```

Comment:
Why not get the result and test that it's correct?

Reply:
Because if it fails it will fail before we get the result, the result is not necessary